src/index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.jpg';
```

Comment:
are those changes for the file-loader package?

Reply:
This file was included as a means of adding static images to the frontend using Typescript. I had searched for ways to test adding images and index.d.ts was one of the ways that I had found. Consequently, these changes were imperative for getting the images to appear on the frontend.